iidm/iidm-tck/src/test/java/com/powsybl/iidm/network/tck/AbstractNetworkTest.java

```diff
@@ -575,7 +575,11 @@ public void testExceptionGetSwitchTerminal2() {
     @Test
     public void testCreate() {
         // check default implementation is used
-        Network.create("test", "test");
+        try {
+            Network.create("test", "test");
+        } catch (Exception e) {
+            fail();
+        }
```

Comment:
It would be cleaner to use `assertDoesNotThrow()` and also check that the result is not null.